src/test/java/mil/dds/anet/test/TestData.java

```diff
@@ -0,0 +1,26 @@
+package mil.dds.anet.test;
+
+import mil.dds.anet.beans.Organization;
+import mil.dds.anet.beans.RollupGraph;
+
+public class TestData {
```

Comment:
This seems quite generic for test data that is used in only one test class. I'd say move the test data to that test class.

Reply:
Disagree - It might only be used in 1 test at the present time, but I would expect that we will require similar code for other tests as we move forward, therefore, we could look to make use of this generic test data class. One could agree this is not an agile approach but I think it is pragmatic.